README.md

```diff
@@ -27,7 +27,7 @@ _Konstantin, please update this section_
 1. Clone repository with submodules: `git clone --recursive https://github.com/MIPT-ILab/mipt-mips.git`
 1. Install LibELF and Boost ([instruction](https://github.com/MIPT-ILab/mipt-mips/wiki/Required-libraries))
 1. [Build MIPS binutils](https://github.com/MIPT-ILab/mipt-mips/wiki/MIPS-binutils) if you need to build MIPS ELF binaries.
-1. In POSIX environment (Linux, OSX, Windows-Ubuntu, and Msys) you can build MIPT-MIPS just by `make mipt-mips` command.
+1. In POSIX environment (Linux, OSX, Windows-Ubuntu, and Msys) create build directory and build MIPT-MIPS just by `cmake <path_to_mipt_mips_directory>/simulator && make` command ([CMake](http://github.com/MIPT-ILab/mipt-mips/wiki/CMake) is required).
 1. WIP on MS Visual Studio support. Currently you can build MIPT-MIPS in command line using `build.cmd` batch file.
```

Comment:
Please explain that IDE users (Visual Studio, Eclipse, CodeBlocks etc.) may use CMake to generate project files.

Reply:
Not very clear what I should write here. Honestly, I dont know how CMake can generate project files in IDE

Reply:
"IDE users (Visual Studio, Eclipse, CodeBlocks etc.) may use CMake to generate project files as well"

Reply:
Usually guys who use IDE know how to set up builds with CMake.